chatblade/parser.py

```diff
@@ -94,6 +94,12 @@ def parse(args):
         help="Stream the incoming text to the terminal",
         action="store_true",
     )
+    parser.add_argument(
+        "--directory",
```

Comment:
This might be better named as `--cache-dir`. I think having the word "cache" in the argument is important, as chatblade uses a couple directories, not just one.